packages/blog/src/utils/image.ts

```diff
@@ -16,5 +16,30 @@ export function getImageUrlFromMedia({
   format = 'large',
   fallback = '/images/WElogo.png',
 }: GetImageUrlFromMediaParams) {
-  return media ? media.formats[format].url : fallback || '/images/WElogo.png'
+  try {
+    if (media) {
+      const preferredFormat = media.formats[format]
+      if (preferredFormat) {
+        return preferredFormat.url
+      } else {
+        //search for the following format available
+        const nextAvailableFormatKeys = Object.keys(media.formats) as FormatType[]
+        if (nextAvailableFormatKeys.length > 0) {
+          nextAvailableFormatKeys.forEach(formatKey => {
+            const nextAvailableFormatData = media.formats[formatKey]
+            if (nextAvailableFormatData) {
+              return nextAvailableFormatData.url
+            }
+          })
```

Comment:
A `return`within a `forEach`does nothing ⚠️

Reply:
thats true, I was thinking on a map. Changing. Thanks!

Reply:
Fixed @bgonp could you check again please?